.github/workflows/build_dealii_master_clang.yml

```diff
@@ -0,0 +1,42 @@
+name: build-dealii-master-clang
+
+on:
+  workflow_dispatch:
+  push:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # trigger at 1:30 am every day
+    - cron:  '30 1 * * *'
+
+concurrency:
+  group: build-tester-base-image
```

Comment:
please use a different concurrency group, otherwise this build may cancel the other deal.II master build